File: PrevozApp/Prevoz/app/src/main/java/com/basic/prevoz/Models/ZahtjevZaPrevozVM.java
package com.basic.prevoz.Models;

import java.util.Date;

/**
 * Created by Developer on 14.07.2017..
 */

public class ZahtjevZaPrevozVM {
    public int Id ;

    public int KorisnikId ;

    public  KorisniciVM Korisnik ;

    public int Status ;
    public int Kolicina;

    public Date DatumKreiranja ;

    public int PrevozId ;

    public  PrevozVM Prevoz ;
}

File: PrevozMVC/ePrevoz.WEB/Models/ZahtjevZaPrevozVM.cs
using ePrevoz.DATA.Helper;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ePrevoz.WEB.Models
{
    public class ZahtjevZaPrevozVM

    {

        public int Id { get; set; }

        public int KorisnikId { get; set; }

        public  KorisnikVM Korisnik { get; set; }

        public int Status { get; set; }

        public int Kolicina { get; set; }

        public DateTime DatumKreiranja { get; set; }

        public int PrevozId { get; set; }

        public  PrevozVM Prevoz { get; set; }
    }
}